### PR TITLE
chore: Fix eslint rule jsdoc/check-values

### PR DESCRIPTION
**Pull request by scinos**
See #56330

#### Changes proposed in this Pull Request

* Fix violations of `jsdoc/check-values`

### CODE REVIEW

File: client/post-editor/media-modal/markup.js [hunk 1]
```diff
@@ -60,9 +60,8 @@ const Markup = {
 	 * Adapted from WordPress.
 	 *
 	 * @copyright 2015 by the WordPress contributors.
-	 * @license See CREDITS.md.
+	 * @license LGPL-2.1
```
Comment (flootr) [inline client/post-editor/media-modal/markup.js:63]: Is it LGPL or GPL? I'm by no means an expert in that area, I just don't find any reference in the WordPress source which says it's the _Lesser_ version of the license.

Comment (scinos) [reply client/post-editor/media-modal/markup.js:63]: I'm not sure. I went to the URL linked in line 64, traversed it up until I found a license.txt (https://github.com/WordPress/WordPress/blob/4.3/wp-includes/js/tinymce/license.txt), and searched for the name in https://spdx.org/licenses/

Comment (flootr) [reply client/post-editor/media-modal/markup.js:63]: Ah, TinyMCE seems to be in fact LGPL. I got confused from the _Adapted from WordPress_ two lines above.

**Review by flootr — APPROVED** [review]
LGTM

**Pull request merged**